website/map/views.py

```diff
@@ -437,14 +437,15 @@ def update_schema():
                         raise APIError("[Sheet %s] Invalid value '%s' in rows[%s], expected empty row" % (sheet['name'], cell_value, rownumber))
                     continue
 
-                hostname = cell_value
+                hostname = cell_value.lower()
 
                 if hostname != "" and not dropOnly:
```

Comment:
Something to think about: `dropOnly` is not at all Pythonic

Reply:
should be `drop_only`?

Reply:
OK, I wlil fix this the next time I touch the dropOnly code